examples/kaggle/kaggle_datasets.py

```diff
@@ -10,7 +10,10 @@
 import time
 import numpy as np
 import deepchem as dc
-from kaggle_features import merck_descriptors 
+try:
+  from kaggle_features import merck_descriptors 
+except ImportError:
+  from kaggle.kaggle_features import merck_descriptors
```

Comment:
Is there any better way to handle this import? Due to that when running benchmark script the working folder is at 'deepchem/examples/', kaggle_features.py can only be imported as kaggle.kaggle_features